warehouse/packaging/models.py

```diff
@@ -148,18 +148,7 @@ def __contains__(self, project):
             return True
 
 
-class TwoFactorRequireable:
-    # Project owner requires 2FA for this project
-    owners_require_2fa: Mapped[bool_false]
-    # PyPI requires 2FA for this project
-    pypi_mandates_2fa: Mapped[bool_false]
-
-    @hybrid_property
-    def two_factor_required(self):
-        return self.owners_require_2fa | self.pypi_mandates_2fa
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏With this removal, I'd expect to see an associated migration in scope dropping the columns. Is that intentionally excluded from this changeset, and will follow?

Reply:
Nope, just forgot to run it: 214fbac